src/components/NewGalaxyModal.js

```diff
@@ -74,61 +82,95 @@ export const NewGalaxyModal = withModalDrawer({ width: 675 })(class NewGalaxyMod
-                ul({ style: { paddingLeft: '1rem', lineHeight: 1.5 } }, [
-                  li({ style: { marginTop: '1rem' } }, [
-                    'Galaxy version xxx'
-                  ]),
-                  li({ style: { marginTop: '1rem' } }, [
-                    'Cloud Compute size of ', span({ style: { fontWeight: 600 } },
-                      [`${getMachineDetails().cpu} CPUS, ${getMachineDetails().memory}  GB of memory, 50 GB disk space`])
-                    //TODO: Define the disk space using DEFAULT_DISK_SIZE from the mob_pd branch
-                  ]),
-                  li({ style: { marginTop: '1rem' } }, [
-                    'Running cloud compute costs $0.19 per hr'
-                  ])
+      ])
+    }
+
+    const renderDeleteWarning = () => {
+      return div({ style: { lineHeight: '22px' } }, [
+        div({ style: { marginTop: '0.5rem' } }, [
+          'Deleting your Cloud Environment will also ',
+          span({ style: { fontWeight: 600 } }, ['delete any files on the associated hard disk']),
+          ' (e.g. results files). Double check that your workflow results were written to ',
+          'the data tab in the workspace before clicking “Delete”'
+        ]),
+        div({ style: { marginTop: '1rem' } }, ['Deleting your Cloud Environment will stop your ',
+          'running Galaxy application and all associated costs. You can create a new Cloud Environment ',
+          'for Galaxy later, which will take 8-10 minutes.'])
+      ])
+    }
+
+    const renderDefaultCase = () => {
+      return h(Fragment, [
+        div({ style: { ...styles.drawerContent, paddingTop: '0.5rem' } }, [
+          div(['Environment consists of an application and cloud compute.']),
+          div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+            div([
+              div({ style: { fontSize: 16, fontWeight: 600 } }, ['Environment Settings']),
+              ul({ style: { paddingLeft: '1rem', lineHeight: 1.5 } }, [
+                li({ style: { marginTop: '1rem' } }, [
+                  'Galaxy version xxx'
                 ]),
-                h(Link, {
-                  ...Utils.newTabLinkProps,
-                  href: ''
-                }, ['Learn more about Galaxy interactive environments.'])
-              ])
+                li({ style: { marginTop: '1rem' } }, [
+                  'Cloud Compute size of ', span({ style: { fontWeight: 600 } },
+                    [`${getMachineDetails().cpu} CPUS, ${getMachineDetails().memory}  GB of memory, 50 GB disk space`])
+                  //TODO: Define the disk space using DEFAULT_DISK_SIZE from the mob_pd branch
+                ]),
+                li({ style: { marginTop: '1rem' } }, [
+                  'Running cloud compute costs $0.19 per hr'
+                ])
+              ]),
+              h(Link, {
+                ...Utils.newTabLinkProps,
+                href: ''
```

Comment:
just so it isn't forgotten could you throw a todo here to get the link for this? (i forgot to do that in my PR)